### ISSUE TITLE
Webstorm IDE - Biome couldn't read the file × /home/<...>/biome.json/rome.json

### ISSUE THREAD

**Issue by appliquant** [biomejs/biome-intellij#4]
Hello,

On Webstorm (Build #WS-233.11799.293, built on December 11, 2023), when I run `alt + shift + L` to format code, I get this error :

```ts
/home/<...>/app/biome.json/rome.json internalError/fs
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 
× Biome couldn't read the file × Biome couldn't read the following file, maybe for permissions reasons or it doesn't exists: /home/<...>/app/biome.json/rome.json
```

I think the plugin is trying to read a `rome.json` file instead of the default `biome.json`.

Here are the steps I followed to install biome : 
- Install Biome `npm install --save-dev --save-exact @biomejs/biome`
- Configure Biome `npx @biomejs/biome init`
-  [Install Biome extension](https://plugins.jetbrains.com/plugin/22761-biome)
- Set up Biome settings in IDE (Languages & Frameworks > Biome Settings)
    - Biome CLI Path : `/home/<...>/app/node_modules/@biomejs/biome/bin/biome`
    - biome.json Directory Path : `/home/<...>/app/biome.json`
    
 Additionaly, running Biome through the terminal `biome format`, `biome check` works fine. 
  
 Here is my `biome.json` file (generated with `npx @biomejs/biome init`) : 

```
{
	"$schema": "https://biomejs.dev/schemas/1.4.1/schema.json",
	"organizeImports": {
		"enabled": true
	},
	"linter": {
		"enabled": true,
		"rules": {
			"recommended": true
		}
	}
}
```

Thank you!

**Comment by appliquant** [comment]
I found a way to get around this error. If Biome settings in IDE (Languages & Frameworks > Biome Settings) are left blank, the `alt + shift + L` runs fine. How ever, as soon as I specify `biome.json Directory Path` to my biome.json config file, I get the error.

**Comment by nhedger** [comment]
Hey @appliquant, this setting expects a **directory**, but `/home/<...>/app/biome.json` is a path to a **file**. Could you try to put `/home/<...>/app` instead?

**Comment by appliquant** [comment]
@nhedger That worked, thanks! For future reference, my Biome settings look like this : 

- Biome CLI Path : `/home/<...>/app/node_modules/@biomejs/biome/bin/biome`
- biome.json Directory Path : `/home/<...>/app` (my biome.json in inside `app` directory`

**Comment by nhedger** [comment]
Glad to hear, I'll close this for now as your immediate issue is solved, and we'll track the improvement in #5.
